### DISCUSSION TITLE
What is the difference between enabling the reactive mode of the Motion_Gen algorithm and not enabling it

### DISCUSSION THREAD

**Discussion by saiyao233** [NVlabs/curobo#277] [Software Q&A] [answered]
I want to know what this mode is mainly used for. From the performance of the demo, there doesn't seem to be a significant difference between the reactive mode and the regular mode. It lacks the dynamic response capability similar to MPC.

**Comment by balakumar-s** [comment, marked as answer]
The difference is in reactive mode is that we don't optimize dt as the robot starts at a non-zero velocity. So you might get trajectories that are not time-optimal.
